pipelines/skylab/optimus/Optimus.changelog.md

```diff
@@ -3,10 +3,11 @@
 
 * Added the latest warp-tools docker to tasks in the Metrics, FastqProcessing and H5adUtils wdls; this incorporates new input parameter for number of output fastq files to fastqprocess
 
-# 6.2.1
-2023-11-20 (Date of Last Commit)
+# 6.2.2
```

Comment:
We should update the Optimus version in the Overview doc

Reply:
thank you! updated it